packages/cli/src/commands/domains/add.ts

```diff
@@ -25,6 +26,14 @@ export default async function add(
   opts: Partial<Options>,
   args: string[]
 ) {
+  const { telemetryEventStore } = client;
+  const telemetry = new DomainsAddTelemetryClient({
+    opts: {
+      store: telemetryEventStore,
+    },
+  });
+
+  telemetry.trackCliFlagForce(opts['--force']);
   const force = opts['--force'];
```

Comment:
Swapping the order of these two lines would help (slightly) with minification.

```suggestion
  const force = opts['--force'];
  telemetry.trackCliFlagForce(force);
```